clustered/client/src/main/java/org/ehcache/clustered/client/internal/service/DefaultClusteringService.java

```diff
@@ -163,6 +163,20 @@ public void start(final ServiceProvider<Service> serviceProvider) {
     }
   }
 
+  @Override
+  public void startForMaintenance(ServiceProvider<MaintainableService> serviceProvider, MaintenanceScope maintenanceScope) {
```

Comment:
Could we leave this method where it was so the changes in the method contents are clearer?